python/pyarrow/tests/test_pandas.py

```diff
@@ -3917,25 +3917,25 @@ def test_metadata_compat_missing_field_name():
                  'name': None,
                  'numpy_type': 'object',
                  'pandas_type': 'mixed-integer'}
-                ],
-             'columns': [
-                 {'metadata': None,
-                  'name': 'a',
-                  'numpy_type': 'int64',
-                  'pandas_type': 'int64'},
-                 {'metadata': None,
-                  'name': 'b',
-                  'numpy_type': 'object',
-                  'pandas_type': 'unicode'}
-                 ],
-             'index_columns': [
-                 {'kind': 'range',
-                  'name': 'qux',
-                  'start': 0,
-                  'step': 2,
-                  'stop': 8}
-                 ],
-             'pandas_version': '0.25.0'}
+            ],
+                'columns': [
```

Comment:
This looks suspicious

Reply:
If you view the diff in split mode, you'll see that it seems ok.

Reply:
Ok, by rearranging the code autopep8 works better.